avm/res/net-app/net-app-account/main.bicep

```
@@ -250,5 +317,109 @@ output resourceGroupName string = resourceGroup().name
 @description('The location the resource was deployed into.')
 output location string = netAppAccount.location
 
-@description('The resource IDs of the volume created in the capacity pool.')
-output volumeResourceId string = (capacityPools != []) ? netAppAccount_capacityPools[0].outputs.volumeResourceId : ''
+@description('The resource IDs of the created capacity pools & their volumes.')
+output capacityPoolResourceIds {
+  resourceId: string
+  volumneResourceIds: string[]
+}[] = [
+  for (capacityPools, index) in (capacityPools ?? []): {
+    resourceId: netAppAccount_capacityPools[index].outputs.resourceId
+    volumneResourceIds: netAppAccount_capacityPools[index].outputs.volumeResourceIds
+  }
+]
+
+// ================ //
+// Definitions      //
+// ================ //
+
+import { backupType } from 'backup-vault/main.bicep'
+@export()
+type backupVaultType = {
```

Missing overall type description for all the defined types as per BCPNFR21 
Ex:
```suggestion
@description('The type for a backup vault.')
type backupVaultType = {
```